src/common/tests/pages/__snapshots__/register.spec.tsx.snap

```diff
@@ -0,0 +1,3 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`Pages/Register test Render correctly 1`] = `<RegisterPage />`;
```

Comment:
by looking this is no guarantee what a change of inside component, so use `mount` for snapshots

Reply:
fix here 2b36b8af941a2ceb43da8923b6e881c0737102bb